src/hidimstat/conditional_feature_importance.py

```diff
@@ -83,7 +87,8 @@ def __init__(
         self.random_state = random_state
 
     def fit(self, X, y=None, groups=None, var_type="auto"):
-        """Fit the imputation models.
```

Comment:
Can't `var_type` be set when the class is instantiated ?

Reply:
`var_type` is linked to the data, there is no sense to define when the class is defined.
In other cases, X and y should be defined in the initialisation.

Reply:
See #377